### ISSUE TITLE
Move Mailer to processor.

### ISSUE THREAD

**Issue by Desertsnowman** [CalderaWP/Caldera-Forms#272]
I hated this right from the beginning. I wanted the mailer to be a form processor.

I'm going to move it now to be that. It annoys me since this means its a contact form builder. and I want it to be a form builder.

**Comment by Shelob9** [comment]
I disagree. Capture entries and database saving is a sensible default that can be disabled. Let's not make something that most users want harder to do.

**Comment by paaljoachim** [comment]
Just as important and creating the form is creating an 
Auto Responder
Email Notification Settings.
These areas need some more discussion as to how to put them in your face so the user does not forget to set them. As it is so easy to forget to setup.

So I would say leave it for now (as in do not change it) and lets have that brain storming session first.
As today creating an auto responder is hidden inside processors (for a user that word does not have much meaning).

Another option:
Mailer is its own section meaning something important happens here...
Email Notification Settings - a good name for any communication that has to do with an email. 
Mailer: DROP DOWN includes admin and user.

Admin opens the existing mailer options.
User opens something very similar to admin but this goes to the user. Creating an auto responder.


This way one creates a separate section for two very important areas that need some extra attention.
Putting it in your face - you got to do this thingie.

One more thing. What about adding a tiny green star below right of the r in Mailer. Similar to how Photoshop creates the star in the title when having edited the psd file. This creates a signal that something is going on here. The user goes to the section does something and saves. The star goes away.

**Comment by paaljoachim** [comment]
I noticed the Insert and update (toggle?) buttons in the latest development branch v1.3.2-b-2. I assume the user can turn off the Mailer section and instead create a Auto Processor for mailer instead. Having the user select which one to use.

Turning off Mailer the message that comes up says "Mailer is currently disabled" - IF an auto processor mailer has been added it would be nice to switch out the message with a link to the Mailer Processor and say something like Go to the Mailer Auto Processor.
